### ISSUE TITLE
Audio files having "Creator data" informations doesn't display correctly

### ISSUE THREAD

**Issue by Sami32** [UniversalMediaServer/UniversalMediaServer#907]
Since i don't feel confortable to explain it in english, a photo will do this much better ;-)
![capture_05312016_173541](https://cloud.githubusercontent.com/assets/18471069/15684513/57e17efc-275f-11e6-9d8a-5d999666ae40.jpg)
I used _Cling Workbench_ because it's more practical for me than go first floor check on the TV, that give the same result anyway.
So as you can see on the right, **all Audio files with "DC: Creator" data doesn't display [No transcoding] and [FFmpeg Audio] infos, but just their title.** ( I tested with MP3, FLAC, OGG, AAC )
Is this minor displaying problem coming from _DLNAResource.java_ ?

PS. Since everybody can easily see it by himself, i am not sure my LOG will be helpfull but if need just ask me.

**Comment by valib** [comment]
@Sami32 please go upstairs and check if the TV is giving the same result that _Cling Workbench_ or check the web interface on your PC. I have also this strange result in _Cling Workbench_ but web interface shows it OK. Unfortunatlelly my TV doesn't support the audio so I can't check it.

**Comment by Sami32** [comment]
@valib  Yes, i got the same result on my Panasonic Viera TV, actually it's there that i saw this for the first time.
Sorry for not being clear in my first post.

I hate Web interface since Firefox is too resources hungry for my old computer...but i just tested after reactivated it in my UMS.conf and effectively it's displaying correctly in the Web interface.

**Comment by valib** [comment]
@Sami32 thanks for the answer. I also test it with Kodi and the result is the same as for the _Cling Workbench_. There is definitely something wrong so I will try to find the problem.

**Comment by Sami32** [comment]
@valib In fact, more testing let me discover that it was specifically the  "Title" field being not empty that give this displaying issue.
String title;
```
		if (
			firstAudioTrack != null &&
			StringUtils.isNotBlank(firstAudioTrack.getSongname()) &&
			getFormat() != null &&
			getFormat().isAudio()
		) {
			title = firstAudioTrack.getSongname() + (player != null && !configurationSpecificToRenderer.isHideEngineNames() ? (" [" + player.name() + "]") : "");
		} else { // Ditlew - org
			title = (isFolder() || subsAreValidForStreaming) ? getDisplayName(null, false) : mediaRenderer.getUseSameExtension(getDisplayName(mediaRenderer, false));
		}

		title = resumeStr(title);
		addXMLTagAndAttribute(
			sb,
			"dc:title",
			encodeXML(mediaRenderer.getDcTitle(title, nameSuffix, this))
		);
```
```
if (player != null) {
			if (isNoName()) {
				displayName = "[" + player.name() + "]";
			} else {
				// Ditlew - WDTV Live don't show durations otherwise, and this is useful for finding the main title
				if (mediaRenderer != null && mediaRenderer.isShowDVDTitleDuration() && media != null && media.getDvdtrack() > 0) {
					nameSuffix += " - " + media.getDurationString();
				}

				if (!configurationSpecificToRenderer.isHideEngineNames()) {
					nameSuffix += " [" + player.name() + "]";
				}
			}
		} else {
			if (isNoName()) {
				displayName = Messages.getString("DLNAResource.0");
				isNamedNoEncoding = true;
				if (subsAreValidForStreaming) {
					isNamedNoEncoding = false;
				}
			} else if (nametruncate > 0) {
				displayName = displayName.substring(0, nametruncate).trim();
			}
		}
```
`isNoName()` condition is not respected with files having "Title".

@SubJunk "Do not add engine names to filenames" is ones of these that you wish turn into a positive form i guess ;-)

**Comment by Sami32** [comment]
Perhaps add an option to disable the _ID3 Tag_ use ?
http://www.universalmediaserver.com/forum/viewtopic.php?f=4&t=8833

**Comment by Nadahar** [comment]
Is disabling ID3 tags really necessary? I mean, is the need big enough to bother? Normally the ID3 tags represent much better info than the file names, and it's loads of software to retag files with bad tags automatically or manuelly.
